src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -65,25 +67,29 @@ const LargeScreenComponent = ({
             alt={`${company} logo`}
             aria-label={`${company} logo`}
             src={partnerLogo}
-            mb={'150px'}
             width={'250px'}
           ></Box>
         </a>
       </Grid>
-      <Grid item pl={'6%'} {...partnerGridStyle} order={contentOrder}>
-        <Typography variant="body1" p={'5% 15% 3% 0'}>
+      <Grid item pl={'6%'} pr={'1%'} {...partnerGridStyle} order={contentOrder}>
```

Comment:
### This whole section on, I'm not sure about this current approach...

There's just so much inconsistency with all the different paddings on different things. It's causing so the text isn't lined up with the logos vertically (when in desktop), the authors have different paddings, etc.

Like visually (besides the vertical alignment) looks like it does the job, but under the hood as a dev - it seems like plug in whatever wherever to get there.

---

I feel like the left `Grid item` (company logo on first `secondaryPartner`) should be vertically and horizontally centered.

Then the right `Grid item` should be the same, but that encompasses all the testimonials and authors it has. This `Grid item` or the `Grid container` itself should have the padding/margin applied for the styling properly.

Then within the right `Grid item` make the spacing between elements as see fit, but should just be `py` or `pt` and `pb` at that point.  Or even better use a `Stack` and `spacing={whatever}` so it's all consistent and easy to determine what the expected behavior/outcome is. Future maintainability.